src/data/set/intervals/basic.lean

```diff
@@ -508,6 +508,9 @@ end
 lemma Iic_singleton_of_bot {a : α} (h_bot : ∀ x, a ≤ x) : Iic a = {a} :=
 @Ici_singleton_of_top (order_dual α) _ a h_bot
 
+lemma Iic_inter_Ioc_of_le {a b c : α} (h : a ≤ c) : Iic a ∩ Ioc b c = Ioc b a :=
+ext $ λ x, ⟨λ H, ⟨H.2.1, H.1⟩, λ H, ⟨H.2, H.1, H.2.trans h⟩⟩
```

Comment:
I think we should try to add the full set here, to try and keep them all next to each other; so that means, add the one with the `ic` swapped, and the variations with `o` and `c`.

Reply:
Oh nevermind, you just moved this lemma